ow_dynamic_terrain/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ project(ow_dynamic_terrain)
 if("$ENV{ROS_DISTRO}" STREQUAL "noetic")
   add_compile_options(-std=c++17)
 endif()
+add_compile_options(-Wno-register)
```

Comment:
The problematic warning this PR seeks to squash only occurs for the noetic build because its using C++17, so wouldn't it make more sense to put this in the if statement?

Reply:
That's weird, since `register` was deprecated in C++11. Did they just not bother to add a warning until 17?